test/directives/prefetch.f90

```diff
@@ -0,0 +1,23 @@
+! RUN: %flang -O1 -S -emit-llvm %s -o - | FileCheck %s --check-prefix=CHECK-PREFETCH
+! RUN: %flang -O1 -S -emit-llvm -Hy,59,0x4 %s -o - | FileCheck %s --check-prefix=CHECK-NOPREFETCH
+
+subroutine prefetch_dir(a1, a2)
+  integer :: a1(4096)
+  integer :: a2(4096)
+
+  do i = 128, (4096 - 128)
+    !$mem prefetch a1, a2(i + 256)
```

Comment:
Is this pgi syntax?

Reply:
Yes. It is already supported by flang1 parser before this patch.